app/views/support_interface/providers/_no_admin_users_warning.html.erb

```diff
@@ -1,6 +1,6 @@
-<% if @provider.sync_courses? && @provider.no_admin_users? %>
-  <div class="govuk-inset-text govuk-!-margin-bottom-4">
+<% if @provider.no_admin_users? %>
```

Comment:
Changed to match same test as used here: https://github.com/DFE-Digital/apply-for-teacher-training/blob/7858f4c5ad0cf4c1207eb66be7295e1c98c5c66f/app/views/support_interface/providers/index.html.erb#L26